docs/guide/index.md

```diff
@@ -61,4 +61,18 @@ messages={{
         message: 'Review',
         id: 'my-events.tab.review'})
     }>
-```
\ No newline at end of file
+```
+
+### Travailler sur une nouvelle branche sur le dépôt original
+
+    git clone git@github.com:lebalz/events-app.git events-app
+    cd events-app
+    git checkout -b "nom-de-la-nouvelle-branche"
+    # Maintenant sur la nouvelle branche
+    # travailler et faire des commits
+    # et lors du premier push
+    git push -u origin neuer-branch-name
+    # continuer à travailler...
+    # pour les push suivants :
+    git push
```

Comment:
Since we use Docusaurus which processes markdown as mdx, code blocks have to be formatted with backticks. The advantage of this: we can specify the language and add code-highlighting for free...

````suggestion
### Travailler sur une nouvelle branche sur le dépôt original
```bash
git clone git@github.com:lebalz/events-app.git events-app
cd events-app
git checkout -b "nom-de-la-nouvelle-branche"
# Maintenant sur la nouvelle branche
# travailler et faire des commits
# et lors du premier push
git push -u origin neuer-branch-name
# continuer à travailler...
# pour les push suivants :
git push
```
````